pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -157,6 +157,10 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 			return errors.New("mongos should be specified")
 		}
 
```

Comment:
can we also add check about configSrv engine (as was asked in ticket)
```
if cr.Spec.Sharding.ConfigsvrReplSet.Storage.Engine != WiredTiger {
    return errors.Errorf("%s storage engine is not supported for config server", cr.Spec.Sharding.ConfigsvrReplSet.Storage.Engine)
}

Reply:
Engine is set only here, in mongod section

Reply:
in such case, can we always force WiredTiger for configSrv replicaset?
don't take value from mongod section?

Reply:
Config Replica set must to use WiredTiger
https://docs.mongodb.com/manual/core/sharded-cluster-config-servers/#replica-set-config-servers
```
To deploy config servers as a replica set, the config servers must run the WiredTiger storage engine.
```